README.md

```diff
@@ -27,28 +36,26 @@ docker build -f Docker/Dockerfile -t testimg . # docker build -f <path/to/Docker
 
 # How to run it
 
-1. Install update
-2. Git clone repo "MyWeb" to /home directory
+1. Git clone repo "MyWeb"
 ```bash
-cd
-git clone ...
+git clone https://github.com/matusik-ops/MyWeb
 ```
 
-3. Install Ansible
+2. Install Ansible
 ```bash
 sudo apt update
 sudo apt install software-properties-common
 sudo add-apt-repository --yes --update ppa:ansible/ansible
 sudo apt install ansible
 ```
 
-4. Rewrite all vars and inventory in automation/Ansible
+3. Rewrite all vars and inventory in automation/Ansible
 ```bash
 cd MyWeb/Ansible
 vim inventory 
 vim ... # vars
 ```
-5. Run playbook in automation/Ansible folder "start.yml"
+4. Run playbook in automation/Ansible folder "start.yml"
```

Comment:
a nice improvement of readme! good job!

overall PR looks good, just please explain why you added static files (CSS, JS, images, ...) into GIT